src/server/game/Entities/Player/Player.cpp

```diff
@@ -21906,14 +21911,9 @@ void Player::UpdateHomebindTime(uint32 time)
 
 void Player::InitPvP()
 {
+    // pvp flag should stay after relog
     if (HasFlag(PLAYER_FLAGS, PLAYER_FLAGS_IN_PVP))
         UpdatePvP(true, true);
-    else if (HasFlag(PLAYER_FLAGS, PLAYER_FLAGS_PVP_TIMER))
```

Comment:
removing not working part, added in 671052575a36624b06379ee5e13a7fea26870194
Not working as bugged in client, mentioned by shauren